examples/src/main/kotlin/examples/inference/saveload/SaveLoadElu.kt

```diff
@@ -0,0 +1,30 @@
+package examples.inference.saveload
+
+import examples.inference.lenet5
+import org.jetbrains.kotlinx.dl.api.core.Sequential
+import org.jetbrains.kotlinx.dl.api.core.activation.Activations
+import org.jetbrains.kotlinx.dl.api.core.loss.Losses
+import org.jetbrains.kotlinx.dl.api.core.metric.Metrics
+import org.jetbrains.kotlinx.dl.api.core.optimizer.SGD
+import org.jetbrains.kotlinx.dl.dataset.mnist
+import java.io.File
+
+
+private const val PATH_TO_MODEL = "savedmodels/elu_lenet_saveload"
```

Comment:
imo it's MODEL_SAVE_PATH as at the begging this file does not exist and it's going to be created after the save process (and then read, but saving is as important as reading)